app/Platform/Actions/ComputeGameSortTitleAction.php

```diff
@@ -26,15 +26,45 @@ public function execute(string $gameTitle): string
      *
      * eg: "Mega Man 10" should not be sorted before "Mega Man 2". With the sort titles
      * set to "mega man 00010" and "mega man 00002" respectively, we can mitigate this.
+     *
+     * Standalone numbers with leading zeroes are padded to 7 digits to preserve significance.
+     * eg: "007" -> "0000007"
+     *
+     * Numbers already padded by replaceRomanNumerals (5 digits) are not given more padding.
+     * eg: "00002" remains "00002"
+     *
+     * Numbers attached to letters are not padded.
+     * e.g: "20Q" -> "20Q", "64th" -> "64th"
```

Comment:
Are you sure this is what we want? I'd expect to find "20Q" between "20 Bottles of Beer" and "21 or Over".

Or using actual titles from the DB:
* 7 Wonders of the Ancient World
* 7th Saga, The
* 8 Eyes

Reply:
There's also an abundance of "3D" games that should be sorted near the "3-D" games, though the latter should probably be normalized to "3D" (not suggesting you do that algorithmically).

Reply:
I think I may have finally gotten it right in latest:

![Screenshot 2024-10-12 at 12 49 27 PM](https://github.com/user-attachments/assets/aedf17c9-2425-4b4e-9683-3efcd120763d)